migrations/0036-add-merch-item-image-table.ts

```diff
@@ -0,0 +1,42 @@
+import {MigrationInterface, QueryRunner, Table, TableIndex } from "typeorm";
+
+const TABLE_NAME = 'MerchItemImages';
```

Comment:
nit: let's call this table `"MerchandiseItemPhotos"`. Since we use "Merchandise" instead of "Merch" for our other table names, we should stay consistent with that. And let's use "photos" instead of "images" since we are using the term "photos" for the environment variables to limit photo size:
![image](https://user-images.githubusercontent.com/33337209/232867032-643bba0f-8b53-453c-8463-f14fe42ce2c2.png)